### PR TITLE
if_not_exist flag to create_hypertable now works on hypertables with …

### PR DESCRIPTION
**Pull request by olofr**
…data as well.

### CODE REVIEW

File: test/sql/create_hypertable.sql [hunk 1]
```diff
@@ -11,6 +11,7 @@ select create_hypertable('test_schema.test_1dim', 'time');
 
 \dt "test_schema".*
 
+insert into test_schema.test_1dim VALUES ('2004-10-19 10:23:54+02',1.0);
```
Comment (RobAtticus) [inline test/sql/create_hypertable.sql]: Can you actually do this afterward the create_hypertable call (and then have a second create_hypertable call)? It'd be nice to test both cases: (1) where the target hypertable is empty and (2) where the target hypertable is not empty.

File: sql/main/ddl.sql [hunk 2]
```diff
@@ -74,6 +75,21 @@ BEGIN
         END IF;
     END IF;
 
+    EXECUTE format('SELECT TRUE FROM _timescaledb_catalog.hypertable WHERE
+                    hypertable.schema_name = %L AND
+                    hypertable.table_name = %L',
+                    schema_name, table_name) INTO is_hypertable;
+
+    IF is_hypertable THEN
```
Comment (cevian) [inline sql/main/ddl.sql]: Should this replace the check in line 118 below. I think we should remove that other check below if it's not necessary anymore.

Comment (olofr) [inline sql/main/ddl.sql]: I don't think so as we don't have a lock here.

File: test/sql/create_hypertable.sql (hunk 1 above)
Comment (olofr) [inline test/sql/create_hypertable.sql]: added

File: test/expected/create_hypertable.out [hunk 3]
```diff
@@ -37,6 +37,7 @@ select create_hypertable('test_schema.test_1dim', 'time');
  test_schema | test_table | table | postgres
 (2 rows)
 
+insert into test_schema.test_1dim VALUES ('2004-10-19 10:23:54+02',1.0);
```
Comment (RobAtticus) [inline test/expected/create_hypertable.out]: I don't think this file was updated to reflect the changes made in the test file? Looks like previous output

File: sql/main/ddl.sql (hunk 2 above)
Comment (cevian) [inline sql/main/ddl.sql]: Ahh ok.

File: test/expected/create_hypertable.out (hunk 3 above)
Comment (RobAtticus) [inline test/expected/create_hypertable.out]: Yep, Travis is failing because of this. Once fixed, you have my approval to merge.

**Pull request merged**